operator-examples/myuser.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: keycloak.org/v1alpha1
+kind: KeycloakUser
+metadata:
+  name: myuser
+  labels:
+    app: sso
+spec:
+  user:
+    username: "myuser"
+    firstName: "John"
+    lastName: "Doe"
+    email: "myuser@keycloak.org"
+    enabled: True
+    emailVerified: False
+    credentials:
+      - type: "password"
+        value: "12345"
```

Comment:
There's something a bit odd about having credentials in clear-text here. It's fine for now, but I think this means we need to consider how/if users are provisioned as CRs.

Reply:
+1 for the future consideration.